src/main/java/refactoring/model/LottoNumber.java

```diff
@@ -0,0 +1,42 @@
+package refactoring.model;
+
+import java.util.Objects;
+
+public class LottoNumber {
+
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+    private final int number;
```

Comment:
```suggestion
    private static final int MAX_LOTTO_NUMBER = 45;
    
    private final int number;
```
https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace

참고하면 좋을 것 같아요